src/nodes.coffee

```diff
@@ -1084,7 +1084,7 @@ exports.PassthroughLiteral = class PassthroughLiteral extends Literal
 
   astProperties: ->
     return {
-      @value
+      value: @originalValue
```

Comment:
Expose the original source version of `PassthroughLiteral` values in the AST as opposed to the "compiled" version that eg unescapes backticks